src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3923,6 +3984,13 @@ static IEnumerable<IEnumerable<SyntaxTrivia>> splitIntoLines(SyntaxTriviaList tr
 
             return new SyntaxTriviaList(syntaxWithoutComments);
         }
+
+        internal override SyntaxNode ParseExpression(string stringToParse)
+            => SyntaxFactory.ParseExpression(stringToParse);
+
+        internal override SyntaxToken CommaTokenWithElasticSpace()
+            => SyntaxFactory.Token(SyntaxKind.CommaToken).WithTrailingTrivia(SyntaxFactory.ElasticSpace);
```

Comment:
this seems extremely feature specific.

Reply:
Moved back to ChangeSignatureService subtypes in the next review.